Codeinwp/feedzy-rss-feeds
# Update error message for invalid feed source

## Summary

Applied the suggestion action from https://github.com/Codeinwp/feedzy-rss-feeds/issues/880#issuecomment-1935856484

## Screenshot

![image](https://github.com/Codeinwp/feedzy-rss-feeds/assets/17597852/2f5a1ce3-8c0c-4390-bb5a-29d909811d11)

## Testing

- Create a new import using a feed that WordPress can not reach. Like: https://www.usnews.com/rss/the-report
- Check if the error message has the updated version.
